plugin/web/extensions/infobox/core/utils/index.ts

```diff
@@ -0,0 +1,15 @@
+import { ActionType } from "../../types";
+
+export function postMsg(action: ActionType, payload?: any) {
+  if (parent === window) return;
+  parent.postMessage({
+    action,
+    payload,
+  });
+}
+
+export function generateId() {
```

Comment:
is this being used anywhere?

Reply:
Not in use actually. I leave it here for primitives, i'm not sure if primitive will get a unique id or not. But seems that it's also okey if no id, so emmm yes maybe better remove it.